### PR TITLE
If propertyValues are not given then the readout should only report d…

### PR DESCRIPTION
**Pull request by nilscb**
…epths.

### CODE REVIEW

File: react/src/lib/components/DeckGLMap/layers/map/terrainMapLayer.ts [hunk 1]
```diff
@@ -227,20 +227,25 @@ export default class TerrainMapLayer extends SimpleMeshLayer<
...
-        );
+        layer_properties.push(createPropertyData("Depth", depth_value));
+        if (!isSeparatePropertyValues) {
+            layer_properties.push(
+                createPropertyData(
+                    "Property",
+                    isNaN(property_value) ? "NaN" : property_value
+                )
+            );
+        }
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/map/terrainMapLayer.ts:248]: I don't think this works if we only show a property.

**Review by hkfb — CHANGES_REQUESTED** [review]

File: react/src/lib/components/DeckGLMap/layers/map/terrainmap.fs.glsl.ts [hunk 2]
```diff
@@ -99,11 +108,11 @@ void main(void) {
    }
 
 
-   bool is_contours = contourReferencePoint != -1.0 && contourInterval != -1.0;
+   bool is_contours = true; //contourReferencePoint != -1.0 && contourInterval != -1.0;
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/map/terrainmap.fs.glsl.ts:111]: Can't disable contours with this change.

Comment (nilscb) [reply react/src/lib/components/DeckGLMap/layers/map/terrainmap.fs.glsl.ts]: I completely forgot to comment this in again!

File: react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx [hunk 3]
```diff
@@ -20,16 +20,18 @@ const valueRange = [-3071, 41048];
 const meshMapLayer = {
     "@@type": "MapLayer",
     id: "mesh-layer",
-    meshUrl: "hugin_depth_5_m.float32",
+    meshUrl:
+        "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/hugin_depth_5_m.float32",
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx:24]: I don't think this change is needed

**Review by hkfb — CHANGES_REQUESTED** [review]

File: react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx (hunk 3 above)
Comment (nilscb) [reply react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx:24]: This is a quite big map and I put it like this to make a more realistic loading time. Seems like it loads quite fast.

File: react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx [hunk 4]
```diff
@@ -28,7 +28,7 @@ const meshMapLayer = {
         rotDeg: 0,
     },
     propertiesUrl: "kh_netmap_5_m.float32",
-    contours: [0, 100.0],
+    contours: [0, 1000],
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx:31]: I think the previous contour interval is more useful.

Comment (nilscb) [reply react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx]: fixed

File: react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx [hunk 5]
```diff
@@ -193,6 +193,46 @@ Readout.parameters = {
     },
 };
 
+export const MapDiscontinous: ComponentStory<typeof DeckGLMap> = () => {
```
[the remark this answers is not in the document]
Comment (nilscb) [reply react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx]: fixed

**Review by hkfb — CHANGES_REQUESTED** [review]
The `MapLayer3dPng` and `MapDiscontinuous` do not display any maps.

When opening the `MapLayer3dPng` story: 

```
mapLayer.ts:279          GET http://localhost:6006/kh_netmap_25_m.png 404 (Not Found)
_callee$ @ mapLayer.ts:279
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
_initialize @ layer.js:760
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:382 Could not load  kh_netmap_25_m.png
_callee$ @ mapLayer.ts:382
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
_initialize @ layer.js:760
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:413 Uncaught (in promise) RangeError: byte length of Float32Array should be a multiple of 4
    at new Float32Array (<anonymous>)
    at _callee$ (mapLayer.ts:413:1)
    at tryCatch (mapLayer.ts:2:1)
    at Generator._invoke (mapLayer.ts:2:1)
    at Generator.next (mapLayer.ts:2:1)
    at asyncGeneratorStep (mapLayer.ts:2:1)
    at _next (mapLayer.ts:2:1)
_callee$ @ mapLayer.ts:413
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
initializeState @ mapLayer.ts:523
_initialize @ layer.js:760
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:279          GET http://localhost:6006/kh_netmap_25_m.png 404 (Not Found)
_callee$ @ mapLayer.ts:279
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
updateState @ mapLayer.ts:558
_update @ layer.js:820
_initialize @ layer.js:773
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:382 Could not load  kh_netmap_25_m.png
_callee$ @ mapLayer.ts:382
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
updateState @ mapLayer.ts:558
_update @ layer.js:820
_initialize @ layer.js:773
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:413 Uncaught (in promise) RangeError: byte length of Float32Array should be a multiple of 4
    at new Float32Array (<anonymous>)
    at _callee$ (mapLayer.ts:413:1)
    at tryCatch (mapLayer.ts:2:1)
    at Generator._invoke (mapLayer.ts:2:1)
    at Generator.next (mapLayer.ts:2:1)
    at asyncGeneratorStep (mapLayer.ts:2:1)
    at _next (mapLayer.ts:2:1)
```

When opening the `MapDiscontinuous` story:

```
mapLayer.ts:279          GET https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/hugin_depth_5_m_w_hole.float32 404
_callee$ @ mapLayer.ts:279
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
_initialize @ layer.js:760
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:309 Could not load  https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/hugin_depth_5_m_w_hole.float32
_callee$ @ mapLayer.ts:309
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
_initialize @ layer.js:760
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:279          GET https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/hugin_depth_5_m_w_hole.float32 404
_callee$ @ mapLayer.ts:279
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
updateState @ mapLayer.ts:558
_update @ layer.js:820
_initialize @ layer.js:773
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:309 Could not load  https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/hugin_depth_5_m_w_hole.float32
_callee$ @ mapLayer.ts:309
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
load_mesh_and_texture @ mapLayer.ts:279
initializeState @ mapLayer.ts:514
updateState @ mapLayer.ts:558
_update @ layer.js:820
_initialize @ layer.js:773
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:339 Uncaught (in promise) RangeError: byte length of Float32Array should be a multiple of 4
    at new Float32Array (<anonymous>)
    at _callee$ (mapLayer.ts:339:1)
    at tryCatch (mapLayer.ts:2:1)
    at Generator._invoke (mapLayer.ts:2:1)
    at Generator.next (mapLayer.ts:2:1)
    at asyncGeneratorStep (mapLayer.ts:2:1)
    at _next (mapLayer.ts:2:1)
_callee$ @ mapLayer.ts:339
tryCatch @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
(anonymous) @ mapLayer.ts:2
asyncGeneratorStep @ mapLayer.ts:2
_next @ mapLayer.ts:2
Promise.then (async)
initializeState @ mapLayer.ts:523
_initialize @ layer.js:760
_initializeLayer @ layer-manager.js:240
_updateSublayersRecursively @ layer-manager.js:209
_updateLayers @ layer-manager.js:174
setLayers @ layer-manager.js:143
updateLayers @ layer-manager.js:151
_onRenderFrame @ deck.js:720
onRender @ animation-loop.js:232
_renderFrame @ animation-loop.js:322
redraw @ animation-loop.js:157
renderFrame @ animation-loop.js:256
requestAnimationFrame (async)
requestAnimationFrame @ request-animation-frame.js:2
_requestAnimationFrame @ animation-loop.js:309
_startLoop @ animation-loop.js:262
(anonymous) @ animation-loop.js:134
Promise.then (async)
start @ animation-loop.js:129
Deck @ deck.js:251
createDeckInstance @ deckgl.js:29
(anonymous) @ deckgl.js:104
invokePassiveEffectCreate @ react-dom.development.js:23487
callCallback @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
flushPassiveEffectsImpl @ react-dom.development.js:23574
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushPassiveEffects @ react-dom.development.js:23447
(anonymous) @ react-dom.development.js:23324
workLoop @ scheduler.development.js:417
flushWork @ scheduler.development.js:390
performWorkUntilDeadline @ scheduler.development.js:157
mapLayer.ts:339 Uncaught (in promise) RangeError: byte length of Float32Array should be a multiple of 4
    at new Float32Array (<anonymous>)
    at _callee$ (mapLayer.ts:339:1)
    at tryCatch (mapLayer.ts:2:1)
    at Generator._invoke (mapLayer.ts:2:1)
    at Generator.next (mapLayer.ts:2:1)
    at asyncGeneratorStep (mapLayer.ts:2:1)
    at _next (mapLayer.ts:2:1)
```

File: react/src/lib/components/DeckGLMap/layers/map/mapLayer.ts [hunk 6]
```diff
@@ -159,7 +160,7 @@ function makeMesh(
     };
 
     // Keep this. Useful info for the user to adjust "maxMeshError" if necessary.
-    console.debug(
+    console.info(
```
Comment (hkfb) [inline react/src/lib/components/DeckGLMap/layers/map/mapLayer.ts:163]: Not in favour of this change. I don't think most people want this information.

**Review by hkfb — APPROVED** [review]

**Pull request merged**